lib/helper.js

```diff
@@ -44,7 +45,7 @@ var validationType = function(value, validation, key){
       return {value: value};
     }else{
       if(types[validation].type) validation = types[validation].type;
-      return {error: key + ': ' + value + ', has to be ' + validation + ' type.', param: key};
+      return {error: key + ': ' + escapeHtml(value) + ', has to be ' + validation + ' type.', param: key};
```

Comment:
alright, if that's the only place, then let's just go with this approach.

I looked at the module but it seems like it hasn't been maintained in a couple years so I guess we can't try to get the maintainer to fix it upstream (as all sails apps using this validator are gonna have the same XSS issue), we'll just have to maintain this as a fork.